crowbar_framework/config/routes.rb

```diff
@@ -99,6 +99,7 @@
   get "crowbar/:controller/1.0/proposals/status(/:id)(/:name)(.:format)", action: "proposal_status", as: "status_proposals_barclamp"
   delete "crowbar/:controller/1.0/proposals/:id(.:format)", action: "proposal_delete", as: "delete_proposal_barclamp"
   delete "crowbar/:controller/1.0/proposals/dequeue/:id(.:format)", action: "proposal_dequeue", as: "dequeue_barclamp"
+  post "crowbar/:controller/1.0/proposals/reset/:id(.:format)", action: "proposal_reset", as: "reset_barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [110/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)